test/core.layoutService.tests.js

```diff
@@ -230,10 +230,10 @@ describe('Test the layout service', function() {
 		expect(chart.scales.xScale1.right).toBeCloseToPixel(512);
 		expect(chart.scales.xScale1.top).toBeCloseToPixel(484);
 
-		expect(chart.scales.xScale2.bottom).toBeCloseToPixel(28);
+		expect(chart.scales.xScale2.bottom).toBeCloseToPixel(60);
```

Comment:
I tried to re-enable the layout service unit tests, and these values was wrong locally. But when running on Travis, many other issues are reported, so finally kept this test disabled.